content/en/docs/api_reference/package-operator-api.md

```
@@ -1191,6 +1397,20 @@ Used in:
 * [Package](#package)
 
 
+### PackageTemplateSpec
+
+PackageTemplateSpec describes the data a package should have when created from a template.
+
+| Field | Description |
+| ----- | ----------- |
+| `metadata` <br>metav1.ObjectMeta | Standard object's metadata.<br>More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata |
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)